index.d.ts

```diff
@@ -129,6 +129,16 @@ export interface TransformAlgorithm {
  *  - {@link SignedXml#checkSignature}
  *  - {@link SignedXml#validationErrors}
  */
+
+/**
+ * @param cert the certificate as a string or array of strings (see https://www.w3.org/TR/2008/REC-xmldsig-core-20080610/#sec-X509Data)
+ * @param prefix an optional namespace alias to be used for the generated XML
+ */
+export interface ComposeKeyInfoContentArgs {
+  cert: string;
+  prefix: string;
+}
```

Comment:
The documentation states string or array of strings, but the parameter only accepts string.